oap-server/server-starter/src/main/resources/ui-initialized-templates/bookkeeper/bookkeeper-cluster.json

```diff
@@ -34,7 +34,7 @@
                     "name": "bookie_ledgers_count"
                   },
                   "expressions": [
-                    "meter_bookkeeper_bookie_ledgers_count"
+                    "latest(aggregate_labels(meter_bookkeeper_bookie_ledgers_count,sum))"
```

Comment:
if the weight type is `Line`, we should not use `latest`. Its result is not a `TIME_SERIES_VALUES`

Reply:
Thanks for the reminder, fixed.